src/components/MapContainer.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react'
+import Map from './Map'
+import { connect } from 'react-redux'
+
+class MapContainer extends React.Component {
+	render() {
+		return (
+			<Map className="map-container" config={this.props.config}/> 
```

Comment:
Can we remove the className here? Or do we need to pass it down from props?

Also, please insert a space between the `}` and `/` character at the end of the line.